packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js

```diff
@@ -1,19 +1,29 @@
 import { useQuery, useQueryClient } from 'react-query';
 import { useFetchClient } from '@strapi/helper-plugin';
+import { stringify } from 'qs';
 
 const QUERY_BASE_KEY = 'review-workflows';
 const API_BASE_URL = '/admin/review-workflows';
 
-export function useReviewWorkflows(workflowId) {
+export function useReviewWorkflows(workflowId, { filters = undefined } = {}) {
```

Comment:
I wonder if passing an object would be better for _all_ the arguments, it would avoid passing `undefined` when you want to pass filters, which i find odd for an API imo.

Reply:
I'm conflicted. For relations this has lead to a huge object which mixes all kinds of domains -- I think separating `id` from the rest makes sense, but I also see the problem with `undefined`.

How would you structure the object?

Reply:
Mh, ok - maybe it doesn't get too bad. I'll apply the same for `useAdminUsers({ id: userId, params: {} }, { reactQueryOptions })` ok?

Reply:
sgtm!

Reply:
Done in https://github.com/strapi/strapi/pull/16820